VideoWeb/VideoWeb/Controllers/TranslationController.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Swashbuckle.AspNetCore.Annotations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace VideoWeb.Controllers
+{
+    [Route("translation")]
+    [ApiController]
+    public class TranslationController : ControllerBase
+    {
+        private readonly ILogger<TranslationController> _logger;
+
+        public TranslationController(
+            ILogger<TranslationController> logger)
+        {
+            _logger = logger;
+        }
+
+        [HttpPost("update")]
+        [SwaggerOperation("updateTranslations")]
+        public async Task<IActionResult> UpdateTranslations()
+        {
+            return Forbid();
+        }
+
+        [HttpGet("{language}")]
+        [AllowAnonymous]
+        [SwaggerOperation("getTranslations")]
+        [Produces("application/json")]
+        public async Task<ActionResult> GetTranslations(string language = "en")
```

Comment:
This will obviously come from a different source. Blob storage or calling another API.